Automattic/jetpack
# Change responses export columns

Second iteration on responses export functionality: reorder and prioritize columns.

Fixes #28677 

## Proposed changes:
Used backend column naming to group and translate. The column order should now be as described here: pcsBup-K4-p2

### Other information:


## Jetpack product discussion
pcsBup-K4-p2

## Does this pull request change what data or activity we track or use?
No

## Testing instructions:
Use a site with some responses or create one, publish a post with a form and fill it a couple of times. Use multiple/different forms for fully testing the behavior.
Go to "Feedback" menu on wp-admin and use the "Export" button. Both CSV or exporting to Google Sheets should show columns:
- Title
- Source
- Date
- [ all columns/values filled on submissions ]
- Consent
- IP Address
